packages/active-campaign-client/src/handlers/manageListSubscription.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { acClient } from '../utils/activeCampaignClient';
+
+export async function manageListSubscription(
+  username: string,
```

Comment:
what is the proper name of this attribute `cognitoId`, `cognitoUserId` or `username`?